runtime/frequency/src/lib.rs

```diff
@@ -242,7 +247,7 @@ impl frame_system::Config for Runtime {
 	/// The maximum length of a block (in bytes).
 	type BlockLength = RuntimeBlockLength;
 	/// This is used as an identifier of the chain. 42 is the generic substrate prefix.
-	type SS58Prefix = SS58Prefix;
+	type SS58Prefix = Ss58Prefix;
```

Comment:
I built mainnet locally and attempted to do a balance transfer just to check that the call filters were acting correctly, but got a SS58Prefix error. I did see that you tested the call filters though. The system constant does show that the prefix is 90 and other configs are as expected.

Reply:
Yes because now that our mainnet specs are files, you will have to change at quite a few places in scripts/onobard etc to make mainnet run locally most importantly paraid to be changed to 2000 and network from polkadot to rococo-local. These changes will make mainnet run locally like rococo 

I did all that and ran a full node locally for mainnet and calls are getting filtered